types/googlepay/index.d.ts

```diff
@@ -1781,13 +1782,6 @@ declare namespace google.payments.api {
          */
         onClick: (event: Event) => void;
 
-        /**
-         * Specifies the button color of the Google Pay button.
-         *
-         * @default "default"
-         */
-        buttonColor?: ButtonColor | undefined;
```

Comment:
This is a breaking API change. Wonder if it would be better to `deprecate` and remove later on?